tests/app/questionnaire/conftest.py

```diff
@@ -1,5 +1,47 @@
+# pylint: disable=redefined-outer-name
 import pytest
 
+from app.data_model.answer_store import AnswerStore
+from app.questionnaire.location import Location
+from app.questionnaire.placeholder_parser import PlaceholderParser
+
+
+@pytest.fixture
+def placeholder_list():
```

Comment:
These guys are only really used in one place, so not sure of the value of having them in the conftest, but it's not a blocking issue.